deploy/terraform-datacommons-website/modules/esp/variables.tf

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  type        = string
+  description = "GCP project id where the API keys will be created."
+}
+
+variable "mixer_grpc_pb_gcs_path" {
+  type        = string
+  description = "Full GCS path to mixer's compiled grpc protobuf definition."
+  default     = "gs://datcom-mixer-grpc/mixer-grpc/mixer-grpc.latest.pb"
```

Comment:
in actual deployment we want to use the versioned grpc_pb file, should that be an variable here?

Reply:
It is already a variable for the module so in the actual example, we would specify the path in .tfvars file, and then specify mixer_grpc_pb_gcs_path=var. mixer_grpc_pb_gcs_path explicitly in the example main.tf file.